cmd/commands/cli/command.go

```diff
@@ -38,6 +39,7 @@ const cliCommandName = "cli"
 // NewCommand constructs CLI based Mysterium UI with possibility to control quiting
 func NewCommand() *cli.Command {
 	var di cmd.Dependencies
+	cmdCLI := &cliApp{}
```

Comment:
Dont need to init:
`var cmdCLI *cliApp`

Reply:
I meant - You should initiate full variable `&cliApp{}`, not only several attributes of it

Reply:
Unfortunately, fields of the `cliApp` depend on the variable that we are getting only from `ctx` in `Action` block.